test/unit/authorizer_test.rb

```diff
@@ -299,18 +299,4 @@ def setup
     refute_includes collection, report1
     assert_includes collection, report2
   end
-
-  test "#find_collection(Host, :permission => :view_hosts, :joined_on: Report, :where => ..) applies where clause" do
```

Comment:
Keep this for Rails 3, IIRC removing this was part of a Rails 4-specific rebase.